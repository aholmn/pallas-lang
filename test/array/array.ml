let () =
  Interpreter.interpreter "../../examples/array"
