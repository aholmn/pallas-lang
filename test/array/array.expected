[1]
[1, 2]
[1, 2, 3]
[1, 1, Strings is also supported, true]
1
3
5
7
RuntimeError: list index out of range
